modules/special-rules/pages/transport-x.adoc

```diff
@@ -1,2 +1,16 @@
 = Transport (X)
 
+{blank}[TODO: Add introductory description.]
+
+---
+
+Where your unit has the {transport} ability:
+
+* The {transport} unit can carry as many Infantry units as we state in brackets.
+For example, the Rhino has {transport-2} -- so you can embark up to two Infantry units in a Rhino, as we explain in the main transport rules.
+
+NOTE: A {transport} unit does not gain the abilities of embarked units.
+For example a {transport} unit that carries a unit with the {save} ability does not gain the {save} ability.
```

Comment:
As with hitch-x.adoc - I don't think we really need the example.
```suggestion
```
Though if we do keep it then I'd like to reword in similar fashion to my comment at hitch-x.adoc